### ISSUE TITLE
Integrate Language Plug-in

### ISSUE THREAD

**Issue by NAMRATA-WOKE** [Green-Software-Foundation/state-of-green-software#197]
**Is your feature request related to a problem? Please describe.**
There are several requests for SOGS to be viewed in non-English languages, such as Japanese, German, Spanish etc. The language plug-in will be an easy automated way for GSF to provide a comprehensive resource in multiple regional languages. The CWG can support by providing/arranging translators to verify translations if required. 

**Describe the solution you'd like**
Language plug-in (paid recommended) - @osamajandali might have some recommendations. @NAMRATA-WOKE recommends GTranslate 

**Describe alternatives you've considered**
Alternative would be to treat every translation request on a case-by-case basis. This will be incredibly time consuming and provide regionally adaptive content much later than when required.

**Additional context**

**Comment by NAMRATA-WOKE** [comment]
@osamajandali any update/thoughts here?

**Comment by zanete** [comment]
@osamajandali - please describe a few possible solutiosn so we can understand what's involved

**Comment by zanete** [comment]
Delayed due to travelling, rolling over to next week

**Comment by osamajandali** [comment]
@NAMRATA-WOKE The best option for what you described is https://www.weglot.com/. it has so many features. GTranslate doesn't have the best experience for websites. Give it a look and let me know what you think

**Comment by NAMRATA-WOKE** [comment]
Yes, familiar with weglot and happy to use that. Any reasons you think we shouldn't from a technical standpoint? @osamajandali
